site/package.json

```diff
@@ -53,10 +53,8 @@
     "mime-types": "^2.1.34",
     "mock-block-dock": "0.0.5",
     "mongodb": "^4.2.2",
-    "next": "12.1.0",
-    "next-connect": "^0.11.0",
-    "next-fonts": "^1.5.1",
-    "next-images": "1.8.1",
+    "next": "^12.1.0",
+    "next-connect": "^0.12.1",
```

Comment:
Using `^` to keep things consistent in this `package.json`. We can drop all `^`s in a separate PR I guess.